packages/server/src/adapters/electron-ipc/rpc-handler.test.ts

```
@@ -0,0 +1,147 @@
+    method: 'POST',
+  }) as any
+
+  const abort_message = await encodeRequestMessage(19, MessageType.ABORT_SIGNAL, undefined) as any
+
+  let sender: any
+  beforeEach(() => {
+    sender = {
+      on: vi.fn(),
+      send: vi.fn(),
+    }
+  })
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)